src/overscroll-color-context.tsx

```diff
@@ -55,8 +55,14 @@ export const OverscrollColorProvider = ({children}: ProviderProps): JSX.Element
 };
 
 const OverscrollColorComponent = () => {
-    const isInverseVariant = useIsInverseVariant();
-    useSetValue(isInverseVariant ? vars.colors.navigationBarBackground : vars.colors.background);
+    const variant = useThemeVariant();
+    useSetValue(
+        variant === 'inverse'
```

Comment:
I'm not a big fan about chaining ternary operators. what about:
```
const getColorFromVariant = (themeVariant: Variant): string => {
    switch (themeVariant) {
        case 'default':
            return vars.colors.navigationBarBackground;
        case 'alternative':
            return vars.colors.backgroundAlternative;
        case 'inverse':
            return vars.colors.navigationBarBackground;
        default:
            const exhaustiveCheck: never = themeVariant;
            throw new Error(`Invalid variant: ${exhaustiveCheck}`);
    }
};

const OverscrollColorComponent = () => {
  const variant = useThemeVariant();
  useSetValue(getColorFromVariant(variant))

  return null;
};
```

Reply:
I also think there's could be a product color gap here for the "backgroundBrandSecondary" that has been used in MOVES as a background color for some sections.

In that case the Overscroll would show up as "backgroundBrand". I don't think it will look broken in the app by any means, as it will mimic the color of the navbar instead of the body, but maybe it's a limitation of the Overscroll to be thought about, that it works with variants but not with the real colors that may acompany those variants, it assumes the color.

Reply:
We've considered the possibility of allowing custom colors for the overscroll, but we concluded that we won't open that door until we have a clear scenario in which the current implementation is not enough.